lib/travis/api/v3/github.rb

```diff
@@ -115,8 +125,6 @@ def info(msg)
       Travis.logger.info(msg)
     end
 
-    private
```

Comment:
I missed that this section was already private back in the `enterprise-2.2` branch so this was unnecessary here.